typescript.html.markdown

```diff
@@ -223,6 +223,30 @@ moreNumbers[5] = 5; // Error, elements are read-only
 moreNumbers.push(5); // Error, no push method (because it mutates array)
 moreNumbers.length = 3; // Error, length is read-only
 numbers = moreNumbers; // Error, mutating methods are missing
+
+// Iterators and Generators
+
+// for..of statement
+// iterate over the list of values on the object being iterated
+let arrayOfAnyType = [1, "string", false];
+for (let val of arrayOfAnyType) {
```

Comment:
If you are not mutating the variable then use `const`

Reply:
Thanks, Done